adflow/pyADflow.py

```diff
@@ -4516,30 +4481,17 @@ def setOption(self, name, value):
         # Set in the correct module
         setattr(module, variable, value)
 
-    def getOption(self, name):
-        # Redefine the getOption def from the base class so we can
-        # make sure the name is lowercase
-
-        if name.lower() in self.defaultOptions:
-            return self.options[name.lower()][1]
-        else:
-            raise Error('%s is not a valid option name.'% name)
-
-    def _getDefOptions(self):
-        """
-        There are many options for ADflow. These technically belong in
-        the __init__ function but it gets far too long so we split
-        them out.
-        """
+    @staticmethod
+    def _getDefaultOptions():
         defOpts = {
             # Input file parameters
             'gridfile':[str, 'default.cgns'],
-            'restartfile':[object, None],
+            'restartfile':[(str, list, type(None)), None],
 
-            # Surface definition parameters:
-            'meshsurfacefamily':[object, None],
-            'designsurfacefamily':[object, None],
-            'closedsurfacefamilies':[object, None],
+            # Surface definition parameters
+            'meshsurfacefamily':[(str, type(None)), None],
```

Comment:
I thought these options take in either None (default), str, or list of string? Do we need to add `list` to the possible types?

Reply:
Maybe @anilyil can clarify this, but for what I gather:
Options with `family` only accept a string or None.
Options with `families` only accept a list or None.

Reply:
Yes I think you are right, options that behave like I described (such as `restartFile`) has been updated already.